pysb/examples/schlogl.py

```diff
@@ -0,0 +1,24 @@
+""" Schlogl model
```

Comment:
You can put the ö back if you put `# -*- coding: utf-8 -*-` at the top of this file. Or, if you want to leave it as pure ASCII, the proper alternative written form is "Schloegl".

Reply:
Thanks! I knew I was missing something trivial.